src/BusinessLogic/DataAccess/GeneralSettings/Entities/GeneralSettings.php

```diff
@@ -61,6 +65,10 @@ public function toArray(): array
             'allowedIPAddresses' => $this->generalSettings->getAllowedIPAddresses(),
             'excludedProducts' => $this->generalSettings->getExcludedProducts(),
             'excludedCategories' => $this->generalSettings->getExcludedCategories(),
+            'enabledForServices' => $this->generalSettings->isEnabledForServices(),
+            'allowFirstServicePaymentDelay' => $this->generalSettings->isAllowFirstServicePaymentDelay(),
+            'allowServiceRegistrationItems' => $this->generalSettings->isAllowServiceRegistrationItems(),
+            'defaultServiceEndDate' => $this->generalSettings->getDefaultServicesEndDate(),
```

Comment:
'defaultServiceEndDate' 
```suggestion
            'defaultServicesEndDate' => $this->generalSettings->getDefaultServicesEndDate(),
```